### ISSUE TITLE
Object.activate is deprecated.

### ISSUE THREAD

**Issue by Eschon** [AtomLinter/linter-jscs#65]
AtomLinter v0.X.Y API has been deprecated. Please refer to the Linter docs to update and the latest API: https://github.com/AtomLinter/Linter/wiki/Migrating-to-the-new-API
```
Object.activate (/Users/egon/.atom/packages/linter/lib/main.coffee:57:8)
Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:240:19)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:221:30)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:165:15)
Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:213:14)
PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:452:21)
```

**Comment by stramel** [comment]
I ran into this issue as well. :+1:

**Comment by cbabos** [comment]
Same here.

**Comment by williamboman** [comment]
Status on this? Is any help wanted?

**Comment by erikdonohoo** [comment]
I deleted all my linter plugins and the linter itself, and then after reinstalling the linter, I was able to install linter-jscs and have it work correctly.

**Comment by n3t0cfg** [comment]
:+1:

**Comment by Johann-S** [comment]
I have the same issue on Windows 8.

**Comment by SpainTrain** [comment]
:+1:

**Comment by rutgerfarry** [comment]
+1

**Comment by malko** [comment]
really need this fixed :+1:

**Comment by abhishekdev** [comment]
:+1:

**Comment by WolfieZero** [comment]
+1

**Comment by patrickburke** [comment]
:+1:

**Comment by DavidOdere** [comment]
+1

**Comment by jacinto123** [comment]
+1 I really hope this gets fixed soon....Is any help wanted?

**Comment by 1000ch** [comment]
This problem will be fixed with #71 :+1:

**Comment by iam4x** [comment]
Fixed with 2.0.0 version.
